policyengine_us/parameters/gov/states/in/tax/income/exemptions/adoption/amount.yaml

```diff
@@ -0,0 +1,12 @@
+description: Indiana provides this additional exemption amount for eligible adopted dependents.
+values:
+  2017-01-01: 3_000
+metadata:
+  unit: currency-USD
+  period: year
+  label: Indiana exemptions additional amount for eligible adopted dependents
+  reference:
+    - title: IC 6-3-1-3.5 "Adjusted gross income" (a)(5)(D)
+      href: http://iga.in.gov/legislative/laws/2021/ic/titles/006#6-3-1-3.5
+    - title: 2023 IT-40 Income Tax Form Instruction Booklet, pg. 26
+      href: https://www.in.gov/dor/tax-forms/2023-individual-income-tax-forms/
```

Comment:
could we add one more tax form reference to show that the value was always 3,000